infrastructure/zk/src/server.ts

```diff
@@ -5,11 +5,7 @@ import fs from 'fs';
 import * as db from './db/db';
 import * as docker from './docker';
 
-import { ethers } from 'ethers';
-
 export async function core(withDocker = false) {
-    prepareForcedExitRequestAccount();
```

Comment:
I'd still use it for dev purposes only. WDYT?

Reply:
updated